readthedocs/config/models.py

```diff
@@ -64,6 +64,11 @@ class BuildJobs(Base):
         "post_build",
     )
 
+    def __init__(self, **kwargs):
```

Comment:
This could use a docstring explaining it, since it's a bit magical. I tried an example: 

```suggestion
    def __init__(self, **kwargs):
    """
    Create an empty list as a default for all possible builds.jobs configs.
    
    This is necessary because it makes the code cleaner when we add items to these lists,
    without having to check for a dict to be created first. 
    """
```